packages/paste-website/src/component-examples/ProgressStepsExamples.ts

```diff
@@ -81,3 +81,66 @@ export const i18nExample = `const ProgressStepsExample = () => {
 };
 
 render(<ProgressStepsExample />);`.trim();
+
+export const additionalContentExample = `const ProgressStepsExample = () => {
```

Comment:
suggestion(blocking): let's remove ProgressStepSeparator from all the docs examples